crates/server/src/graphql/modules/post/mutation/comment_create.rs

```diff
@@ -0,0 +1,52 @@
+use async_graphql::{Context, InputObject, Result, SimpleObject};
+use playa::post::service::CreatePostDto;
+use pxid::graphql::Pxid;
+use serde::{Deserialize, Serialize};
+
+use crate::{
+    context::SharedContext,
+    graphql::modules::post::types::{Post, PostError},
+    services::auth::Token,
+};
+
+#[derive(Debug, InputObject)]
+pub struct CommentCreateInput {
+    pub content: String,
+    pub parent_id: Pxid,
+}
+
+#[derive(Debug, Deserialize, Serialize, SimpleObject)]
+pub struct CommentCreate {
+    comment: Option<Post>,
+    error: Option<PostError>,
+}
+
+impl CommentCreate {
+    pub async fn exec(ctx: &Context<'_>, input: CommentCreateInput) -> Result<Self> {
+        let context = ctx.data_unchecked::<SharedContext>();
+        let token = ctx.data_unchecked::<Token>();
+
+        let claims = context.services.auth.verify_token(token).unwrap();
+
+        let dto = CreatePostDto {
+            author_id: claims.uid,
+            parent_id: Some(input.parent_id.into_inner()),
+            title: "".to_string(),
```

Comment:
I think we should have a separate service method to create comments.
A new method with its own `Dto` and validation.